### ISSUE TITLE
Multiselect actions

### ISSUE THREAD

**Issue by Mikescops** [nextcloud/photos#83]
A user has to be able to apply a certain action (delete, add to album) by selecting a batch of photos from the timeline.

There a different ways to handle that, on of them is to display a check mark when hovering the thumbnail and a click on the icon leads the user to the select mode where he can click on thumbnails to add them to the selection and then apply an action to the files from the menu.

Here is a more visual example from Google Photos:
![image](https://user-images.githubusercontent.com/4266283/71664718-508dd380-2d5a-11ea-9b85-2cd1eb590782.png)

**Comment by skjnldsv** [comment]
cc @jancborchardt

**Comment by jancborchardt** [comment]
Yep, multiselect and multiselect actions should absolutely be possible.

For the first step, what you describe above works:

> display a check mark when hovering the thumbnail and a click on the icon leads the user to the select mode where he can click on thumbnails to add them to the selection and then apply an action to the files from the menu.

We then also need to have a way to enter the select mode when you work on a touchscreen. It could be in a 3-dot menu next to the "Share" icon next to the folder heading.

**Comment by jakobroehrl** [comment]
@skjnldsv How hard is this to do? Maybe I could do it with a little help from you?

**Comment by skjnldsv** [comment]
@jakobroehrl sure thing! I don't think it's that complicated to manage for the selection
But we're lacking a proper standard to register file actions. So we'd have no way of knowing what we're allowed to do with those files unless there is one list of actions hardcoded in the app.

It would be a nice first step anyway, but it will needs to be hardcoded and when the proper standard arrive, we'd be able to use it. Are you still up for the task? :)

**Comment by kanintesova** [comment]
I notice this is implemented in the mobile phone app, atleast on iOS. But no such thing on the desktop web browser.

**Comment by RikkiBC** [comment]
Having to redirect web users from Photos view to File view is a little frustrating for some users, especially those coming from Google Photos (as per the original post in here). Is there any update on this issue? 
Not trying to be rude, just wondering.

**Comment by r01k** [comment]
3 years later and the luxury to select multiple photos is not yet present.

**Comment by joshtrichards** [comment]
This was implemented awhile ago. I believe in #1119.
